src/axes.jl

```diff
@@ -235,6 +235,17 @@ function get_ticks(sp::Subplot, axis::Axis; update = true)
     return axis.plotattributes[:optimized_ticks]
 end
 
+# Ticks getter functions
+get_ticks(sp::Subplot, s::Symbol) = get_ticks(sp, sp[Symbol(s, :axis)])
+xticks(sp::Subplot) = get_ticks(sp, :x)
+yticks(sp::Subplot) = get_ticks(sp, :y)
+zticks(sp::Subplot) = get_ticks(sp, :z)
+get_ticks(p::Plot, s::Symbol) = [get_ticks(sp, s) for sp in p.subplots]
+xticks(p::Plot) = get_ticks(p, :x)
+yticks(p::Plot) = get_ticks(p, :y)
+zticks(p::Plot) = get_ticks(p, :z)
+export xticks, yticks, zticks
```

Comment:
```suggestion
```
There is no official guideline on this, but as far as I can see, we don't export "getters" (like `Plots.xlims` for example)

Reply:
Maybe I'm misunderstanding something, but isn't `xlims` exported there?

https://github.com/JuliaPlots/Plots.jl/blob/1ceb3a561f5e9f20471636865180d603d09753ed/src/Plots.jl#L68-L70

Maybe it's just a matter of moving these exports in that long list?

Reply:
Yep, this works for me with the current version (v1.11.2):
```julia
julia> using Plots

julia> xlims(plot())
(0.0, 1.0)
```

Reply:
good Point, I didn't know. There is nothing to be seen here then ^^;

Reply:
But should I move the export of `xticks` to where that of `xlims` is?